android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/ListViewHolder.java

```diff
@@ -203,8 +221,8 @@ public void bind(final ListItemProxy proxy, final boolean selected)
 					}
 
 					// Support selected backgrounds.
-					nativeView.setBackground(generateSelectedDrawable(properties, backgroundDrawable));
-					borderView.setActivated(selected);
+					this.container.setBackground(generateSelectedDrawable(properties, backgroundDrawable));
+					this.container.setActivated(selected);
```

Comment:
I could be wrong, but I **think** you need to move the `setActivated()` method call after/outside the `if (nativeView != null) {} block. The reason may be because of the item recycling changes I made in 10.0.1.

If you enable edit mode, scroll to the bottom of a long list, check the last item, scroll back to the top, disable edit mode, re-enable edit mode, and then scroll back to the bottom... you'll find that the last item is still wrongly checked. (All items should be unchecked when enabling edit mode; it's supposed to reset it which works fine for items on-screen.)